integrating-melodi-analytics/react-component-new.mdx

```diff
@@ -0,0 +1,173 @@
+        type: 'markdown'
+      },
+      {
+        role: 'assistant',
+        content: aiGeneratedText,
+        type: 'markdown'
+      }
+    ],
+    projectName: 'My Project',
+    metadata: {
+      environment: 'production',
+      feature: 'chat'
+    }
+  }}
+  userInfo={{
+    id: user.id,
+    email: user.email
+  }}
+/>
+```
+
+5. See feedback as it comes into your [Feedback Inbox](https://app.melodi.fyi/admin/feedback-inbox)
+
+## Usage Examples
+
+### Basic Usage
+```jsx
+<MelodiFeedbackWidget
+  thread={{
+    messages: [
+      {
+        role: 'user',
+        content: userSubmittedText,
+        type: 'markdown'
+      },
+      {
+        role: 'assistant',
+        content: aiGeneratedText,
+        type: 'markdown'
+      }
+    ],
+    projectName: 'My Project',
+    metadata: {
+      environment: 'production',
+      feature: 'chat'
+    }
+  }}
+  userInfo={{
+    id: user.id,
+    email: user.email
+  }}
+/>
+```
+
+### Advanced Usage with JSON Messages
+```jsx
+<MelodiFeedbackWidget
+  thread={{
+    messages: [
+      {
+        role: 'user',
+        content: 'Tell me a joke',
+        type: 'markdown'
+      },
+      {
+        role: 'json_tool',
+        type: 'json',
+        jsonContent: {
+          joke_setup: 'Why did the chicken cross the road?',
+          joke_punchline: 'To get to the other side!'
+        },
+        metadata: {
+          source: 'jokes_database'
+        }
+      },
+      {
+        role: 'assistant',
+        content: 'Why did the chicken cross the road? To get to the other side!',
+        type: 'markdown'
+      }
+    ],
+    projectName: 'Joke Bot',
+    externalId: 'joke-conversation-123',
+    metadata: {
+      environment: 'production',
+      feature: 'joke_generator',
+      team: 'content'
+    }
+  }}
+  userInfo={{
+    id: 'user-123',
+    email: 'user@example.com'
+  }}
+  companyName="My Company"
+/>
+```
+
+## Component Props
+
+### MelodiFeedbackWidget Props
+
+| Prop | Type | Required | Description |
+|------|------|----------|-------------|
+| `thread` | ThreadProps | Yes | Configuration for the conversation thread |
+| `userInfo` | UserInfo | No | Information about the current user |
+| `companyName` | string | No | Your company name to display in the feedback popover |
+
+### Message Types
+
+Messages in a thread can be one of three types:
+
+1. **Markdown Messages** (default)
+   - Used for standard text content
+   - Set `type: 'markdown'`
+   - Content goes in the `content` field
+
+2. **JSON Messages**
+   - Used for structured data or tool outputs
+   - Set `type: 'json'`
+   - Data goes in the `jsonContent` field
+   - Requires `role: 'json_tool'`
+
+3. **Message Roles**
+   - `user`: Messages from the end user
+   - `assistant`: AI-generated responses
+   - `json_tool`: Structured data or tool outputs
+
+## Visual Example
+
+Here's how your thread will appear in the Melodi dashboard:
+
+![Thread Render](../images/render-thread.png)
```

Comment:
Just a suggestion, but I think it would be nice if the image here matched the examples above

Reply:
Good point. I'm going to update the examples here to match the standard example we're using everywhere and then add an updated image.